data/map.txt

```diff
@@ -31489,7 +31489,8 @@ planet Solace
 	landscape land/valley6
 	description `Solace is a fertile world that was already home to Earth-like vegetation and animals even before the first settlers arrived. It has now been inhabited for nearly half a millennium, and sprawling cities cover much of its surface.`
 	description `	In addition to being one of the major metropolitan centers of the galactic South, Solace is best known as the home of Delta V Corporation, makers of plasma engines and rockets of various sorts.`
-	spaceport `The spaceport on Solace serves not only for incoming and outgoing commerce, but also as a testing ground for the next generation of Delta V technology. On your way around the port you stop to watch some tests of engines so powerful that the ground shakes beneath them.`
+	spaceport `Solace's spaceport is located in a satellite town of one of its major cities. The town itself is an extensive industrial park, stretching out for some distance. Eventually the warehouses and factories give way to valleys and farmlands.` 
+	spaceport `	The spaceport serves not only for incoming and outgoing commerce, but also as a testing ground for the next generation of Delta V technology. On your way around the port you stop to watch some tests of engines so powerful that the ground shakes beneath them.`
```

Comment:
Generally when something like this grows to encroach on farmland, you  have very developed industrial things that simply add on beyond the previous one at the nearest available distance. Which means you go from industrial park to farmland very quickly, often the farmland starts metres beyond the fenceline of the last warehouse.
```suggestion
	spaceport `Solace's spaceport is located in a satellite town of one of its major cities. The town itself is an extensive industrial park, stretching out for some distance where the warehouses and factories abruptly give way to valleys and farmlands.` 
	spaceport `	The spaceport serves not only for incoming and outgoing commerce, but also as a testing ground for the next generation of Delta V technology. On your way around the port you stop to watch some tests of engines so powerful that the ground shakes beneath them.`
```

Reply:
I see your point, but I think they both mean the same thing in the end. I'll change it to something else that makes it more clear anyway.